### PR TITLE
fix symbol display

### PR DESCRIPTION
**Pull request by davidgohel**
It should fix symbol display issue

    funplot <- function(){
      plot(c(0,4), c(5,1))
      text(1, 1, "universal", adj = 0); text(2.5, 1,  "\\042")
      text(3, 1, expression(symbol("\042")))
      text(1, 2, "existential", adj = 0); text(2.5, 2,  "\\044")
      text(3, 2, expression(symbol("\044")))
      text(1, 3, "suchthat", adj = 0); text(2.5, 3,  "\\047")
      text(3, 3, expression(symbol("\047")))
      text(1, 4, "therefore", adj = 0); text(2.5, 4,  "\\134")
      text(3, 4, expression(symbol("\134")))
      text(1, 5, "perpendicular", adj = 0); text(2.5, 5,  "\\136")
      text(3, 5, expression(symbol("\136")))
    }
    
    if (require("htmltools")) {
      htmlSVG(funplot())
    }

### CODE REVIEW

**Comment by hadley** [comment]
Would you mind making a unit test or two?

**Comment by davidgohel** [comment]
ops.
Yes I will

File: src/devSVG.cpp [hunk 1]
```diff
@@ -301,6 +302,39 @@ static void svg_size(double *left, double *right, double *bottom, double *top,
...
+    raster_[i] = raster[i] ;
+  }
+
+  std::string base64_str = gdtools::raster_to_str(raster_, w, h, width, height, (Rboolean) interpolate);
+
+  fprintf(svgd->file, "<image x='%.2f' y='%.2f' ", x, y - height );
+  fprintf(svgd->file, "width='%.2f' height='%.2f' ", width, height);
+
+  if (fabs(rot)>0.001) {
+    fprintf(svgd->file, "transform='rotate(%0.0f,%0.0f,%0.0f)' ", -1.0 * rot, x, y );
```
Comment (hadley) [inline src/devSVG.cpp]: Would you mind doing the same thing as for text? It makes it a bit easier for me to understand

Comment (davidgohel) [inline src/devSVG.cpp]: You mean `if (rot == 0)`?

Comment (hadley) [inline src/devSVG.cpp]: Yeah - i.e. use x and y if rot == 0, otherwise use transform + rotate

**Comment by davidgohel** [comment]
This version should be ok.
 
I am using only `rotate` when `rot != 0`.

**Comment by hadley** [comment]
Could you please also add a bullet to NEWS?

**Comment by hadley** [comment]
Two things:

* Could you please add a bullet point to NEWS?

* When running the tests I get a warning: "In doc_parse_file(con, encoding = encoding, as_html = as_html) : Namespace prefix xlink for href on image is not defined [201]" (you might not see that because the released version of testthat silences warnings). I tried the inserting ns ref before the xlink, but that didn't fix the problem - I think it might need to be inserted on the parent element. So maybe just always insert in `svg_new_page()`?

**Comment by davidgohel** [comment]
OK, I can see the warning you're talking about.

I don't know yet where to properly add the xlink namespace. I will have a look later today...

**Comment by davidgohel** [comment]
I don't understand why we get that warning. 

According to the [spec](http://www.w3.org/TR/SVG/struct.html#ImageElementHrefAttribute), the namespace prefix should be declared in the `svg` tag

**Comment by hadley** [comment]
You're currently declaring it on the image tag?

**Comment by davidgohel** [comment]
I moved it in the SVG tag.

**Comment by hadley** [comment]
The problem was that `xmlSVG()` needed to pass along the standalone flag, and then you also have to do a bit more work once the xml doc has namespaces. Looks good now.

**Comment by davidgohel** [comment]
Cool

**Pull request merged**